### PR TITLE
OBSDOCS-1084: Update Default monitoring targets section with missing …

### PR DESCRIPTION
**Pull request by eromanova97**
Version(s): `enterprise-4.12` and later

Issue: [OBSDOCS-1084](https://issues.redhat.com/browse/OBSDOCS-1084)

Links to docs preview: 

* OCP: [Default monitoring targets
](https://77351--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/monitoring-overview.html#default-monitoring-targets_monitoring-overview)
* OSD: [Default monitoring targets
](https://77351--ocpdocs-pr.netlify.app/openshift-dedicated/latest/observability/monitoring/monitoring-overview.html#default-monitoring-targets_monitoring-overview)
* ROSA: [Default monitoring targets
](https://77351--ocpdocs-pr.netlify.app/openshift-rosa/latest/observability/monitoring/monitoring-overview.html#default-monitoring-targets_monitoring-overview)

QE review:
- [x] QE has approved this change.

**Additional information:**

### CODE REVIEW

File: modules/monitoring-default-monitoring-targets.adoc [hunk 1]
```diff
@@ -7,17 +7,17 @@
...
 ifndef::openshift-dedicated,openshift-rosa[]
-In addition to the components of the stack itself, the default monitoring stack monitors:
+In addition to the components of the stack itself, the default monitoring stack monitors additional platform components.
+
+The following are examples of monitoring targets:
 endif::openshift-dedicated,openshift-rosa[]
 
 ifdef::openshift-dedicated,openshift-rosa[]
-Red Hat Site Reliability Engineers (SRE) monitor the following platform targets in your {product-title} cluster:
+The following are examples of targets monitored by Red Hat Site Reliability Engineers (SRE) in your {product-title} cluster:
```
[the remark this answers is not in the document]
Comment (eromanova97) [reply modules/monitoring-default-monitoring-targets.adoc:16]: @simonpasquier so the ones that should be removed for the ROSA are:
* OpenShift API server
* OpenShift Controller Manager
* Operator Lifecycle Manager (OLM)

Do I understand correctly?

[the remark this answers is not in the document]
Comment (simonpasquier) [reply modules/monitoring-default-monitoring-targets.adoc:16]: The first 2 at least. Not sure for OLM but probably yes.

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Comment by eromanova97** [comment]
I see the cherry-picking beautifully failed :smile: i will let you know once I fix it

**Comment by eromanova97** [comment]
@skopacz1 I am done with manual CPs :+1:

**Pull request merged**